generator/src/main/java/org/corfudb/generator/Generator.java

```diff
@@ -0,0 +1,75 @@
+package org.corfudb.generator;
+
+import java.util.List;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.Future;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.TimeUnit;
+
+import org.corfudb.generator.operations.CheckpointOperation;
+import org.corfudb.generator.operations.Operation;
+import org.corfudb.generator.operations.RemoveOperation;
+import org.corfudb.runtime.CorfuRuntime;
+import org.corfudb.runtime.exceptions.NetworkException;
+import org.corfudb.runtime.exceptions.TransactionAbortedException;
+import org.corfudb.runtime.view.stream.IStreamView;
+
+import static java.util.concurrent.Executors.newWorkStealingPool;
+
+/**
+ * Created by maithem on 7/14/17.
+ */
+public class Generator {
+    public static void main(String[] args) {
+        String endPoint;
+        int numStreams;
+        int numKeys;
+        long cpPeriod;
+        int numThreads = 8;
+        int numOperations;
+
+        CorfuRuntime rt = new CorfuRuntime("localhost:9000").connect();
+
+        State state = new State(50, 100, rt);
+
+        Runnable cpTrimTask = () -> {
+            Operation op = new CheckpointOperation(state);
+            op.execute();
+        };
+
+        final ScheduledExecutorService scheduler = Executors.newScheduledThreadPool(1);
+        scheduler.scheduleAtFixedRate(cpTrimTask, 30, 20, TimeUnit.SECONDS);
+
+        ExecutorService appWorkers = newWorkStealingPool(numThreads);
+
+        Runnable app = () -> {
+            List<Operation> operations = state.getOperations().sample(3000000);
+            for (Operation operation : operations) {
+                try {
+                    operation.execute();
+                } catch (IndexOutOfBoundsException ex) {
+                    ex.printStackTrace();
+                    System.exit(-1);
+                }catch
+                (TransactionAbortedException | NetworkException e) {
+                    System.out.println(e);
+                }
+            }
+        };
+
+        Future[] appsFutures = new Future[numThreads];
+
+        for (int x = 0; x < numThreads; x++) {
+            appsFutures[x] = appWorkers.submit(app);
+        }
+
+        for (int x = 0; x < numThreads; x++) {
+            try {
+                appsFutures[x].get();
+            } catch (Exception e) {
+                System.out.println("App Exception: " + e);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)